PULL-REQUEST-TEMPLATE.md

```diff
@@ -0,0 +1,49 @@
+# PR Template
+
+This page list recommendations and general rules, so we can keep a clean and consistent PR description.
+
+*The template starts after the **horizontal line below***
+
+### References
+- https://embeddedartistry.com/blog/2017/08/04/a-github-pull-request-template-for-your-projects/
+
+---
```

Comment:
delete this stuff from the template?

Reply:
Why not, I was'nt sure if it made sense to keep it!